src/commands/init.ts

```diff
@@ -62,11 +63,26 @@ export abstract class InitCommand extends BaseCommand {
     }
     this.log(`Start the process to create your project '${project_name}'.`);
     const selections = await ProjectUtils.getSelections();
+
+    this.log(`                                                   7###        s##`);
+    this.log(`    ;####^####           ,###Q                      @##        7%"  ]##b                     @##p`);
+    this.log(`   {####  ####           # @##p    W##pa##  ,s#WW#m @##,###m  *### W@##MWm ,sMWWm,   ,m#WW#pW###MWL`);
+    this.log(`  #####      &####      @\`  @##    j##b^7Y ###    ^ @##  '###  @##~ @##b  ###  '##b ###    \` @##b`);
+    this.log(` WWWWQ,,,   #####      @WWWWW###   j##b   j###      @##   ###  @##~ @##b ]###"T"TTb]###      @##b`);
+    this.log(`     @###b ####b      ]#     @##Q  j##b    @##p   , @##   ###  @##~ @##b  @##p    ,^@##    , @##b`);
+    this.log(`     %888W8888"      *WWW    4WWWW %WWW=    7%W#MT *WWWW 4WWW=*WWWW  %W#W^ ^%WWWWb   7%WWW"   %WWW^\n`);
```

Comment:
I really love this logo. My only concern is the case where the user is using a small terminal screen.

Reply:
I agree. I'll go a head and switch back to the original header

Reply:
updated logo to one more compatible with terminal screens less than 35px by 134px in size (eg. visual studio console integration)